newIDE/app/src/UI/EditorMosaic/index.js

```diff
@@ -19,7 +18,6 @@ export type Editor = {|
   renderEditor: () => React.Node,
   noTitleBar?: boolean,
   title?: React.Node,
```

Comment:
This should be now `title?: MessageDescriptor`,

Reply:
Oh yes, silly mistake :)